+adi/+AD7124_8/Rx.m

```diff
@@ -13,9 +13,9 @@
 
     
     properties (Nontunable, Hidden)
-        channel_names = {'voltage0','voltage1','voltage2','voltage3',...
-            'voltage4','voltage5','voltage6','voltage7','voltage8','voltage9',...
-            'voltage10','voltage11','voltage12','voltage13','voltage14','voltage15'};
+        channel_names = {'voltage0-voltage1','voltage2-voltage3',...
```

Comment:
well, it's how the implementation of the AD7124 driver is, in the Linux IIO framework;
since the inputs on the AD7124 are differential, the IIO framework decided to represented this as 2 channel-pairs (i.e. voltage0-voltage1 and so on);  this is how they show up, when a channel is defined as `differential`;

what puzzles me, is why this was represented as `voltage0','voltage1, etc` in the first place

the AD7124 driver has been implemented in Linux (as differential) for [a number of years](https://github.com/analogdevicesinc/linux/commit/b3af341bbd9662c672d596427ad23973584221d3#diff-20dbeef81fb2f3c84c5d3cbc046ca2cd1c1470f03d64e376e89956b2359af726R127);
so, the channels should have looked more like `'voltage0-voltage1','voltage2-voltage3',`

unless, the no-OS implementation was used to test this initially;